modules/invoice/model/invoice.class.php

```diff
@@ -1326,5 +1328,143 @@ function CheckIfAnythingChanged($InvoiceID, $InvoiceLineIDs, &$args) {
+
+      // Missing info! Using company's address and contact info.
+      $altinn_invoice->AccountingCustomerParty->Party->PostalAddress->StreetName = $_lib['sess']->get_companydef('IAddress');
+      $altinn_invoice->AccountingCustomerParty->Party->PostalAddress->BuildingNumber = '';
+      $altinn_invoice->AccountingCustomerParty->Party->PostalAddress->CityName = $_lib['sess']->get_companydef('ICity');
+      $altinn_invoice->AccountingCustomerParty->Party->PostalAddress->PostalZone = $_lib['sess']->get_companydef('IZipCode'); // ZipCode
+      $altinn_invoice->AccountingCustomerParty->Party->PostalAddress->Country->IdentificationCode = $_lib['sess']->get_companydef('ICountryCode');
+      $altinn_invoice->AccountingCustomerParty->Party->Contact->Telephone = $_lib['sess']->get_companydef('Phone');
+      $altinn_invoice->AccountingCustomerParty->Party->Contact->Mobile = $_lib['sess']->get_companydef('Mobile');
+      $altinn_invoice->AccountingCustomerParty->Party->Contact->Telefax = $_lib['sess']->get_companydef('Fax');
+      $altinn_invoice->AccountingCustomerParty->Party->Contact->ElectronicMail = $_lib['sess']->get_companydef('Email');
+
+      // Invoice payment means
+      $altinn_invoice->PaymentMeans->PaymentMeansCode = 42;
+      $altinn_invoice->PaymentMeans->PaymentDueDate = $due_date;
+      $altinn_invoice->PaymentMeans->PayeeFinancialAccount->ID = $bank_account_number;
+      $altinn_invoice->PaymentMeans->PayeeFinancialAccount->Name = 'Bank';
+      // Missing info! No way to determine which of the accounts is the bank account used for tax. 
+      // $altinn_invoice->PaymentMeans->PayerFinancialAccount->ID = $invoice->BankAccount;
+      // $altinn_invoice->PaymentMeans->PayerFinancialAccount->Name  = 'Bank';
+
+      $altinn_invoice->PaymentMeans->InstructionID = $kid;
+      $altinn_invoice->PaymentMeans->InstructionNote = "KID";
+
+      // Invoice lines
+      $taxH[0]->TaxableAmount = $amount;
+      $taxH[0]->TaxAmount = 0;
+
+      $altinn_invoice->InvoiceLine[0]->ID = 0;
+      $altinn_invoice->InvoiceLine[0]->LineExtensionAmount = $amount;
+      $altinn_invoice->InvoiceLine[0]->TaxTotal->TaxAmount = 0;
+      $altinn_invoice->InvoiceLine[0]->TaxTotal->TaxSubtotal->TaxableAmount = $amount;
+      $altinn_invoice->InvoiceLine[0]->TaxTotal->TaxSubtotal->TaxAmount = 0;
+      $altinn_invoice->InvoiceLine[0]->TaxTotal->TaxSubtotal->Percent = 0;
+      $altinn_invoice->InvoiceLine[0]->TaxTotal->TaxSubtotal->TaxCategory->TaxScheme->ID = 'VAT';
+
+      $altinn_invoice->InvoiceLine[0]->Item->Name = $invoice_type . " TAX";
+      $altinn_invoice->InvoiceLine[0]->Item->Description = $invoice_type . " TAX for " . $periode;
+
+      $altinn_invoice->InvoiceLine[0]->Price->PriceAmount = $amount;
+      $altinn_invoice->InvoiceLine[0]->Price->BaseQuantity = 1;
+
+      $altinn_invoice->TaxTotal['TaxAmount'] = 0;
+
+      foreach($taxH as $VatPercent => $vat) {
```

Comment:
Maybe directly hardcode the values from params here since we only have for 0% tax group.